incubator/hnc/api/v1alpha1/hierarchy_types.go

```diff
@@ -145,6 +149,52 @@ type AffectedObject struct {
 	Name      string `json:"name,omitempty"`
 }
 
+func NewAffectedNamespace(ns string) AffectedObject {
+	return AffectedObject{
+		Version: "v1",
+		Kind:    "Namespace",
+		Name:    ns,
+	}
```

Comment:
Just confirming, is this a change on how to represent namespaces? Previously, we planned to use only the name field to represent namespaces, right?

This may be better in that we don't need to specifically document the representation.

Reply:
At the API level, we always represented namespaces this way - which is the same way that Kubernetes itself represents namespaces. Internally, we didn't, but now we're using the same representation internally and externally.

Reply:
Thanks for the explanation!